src/main/java/uk/gov/companieshouse/itemhandler/environment/EnvironmentVariables.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.companieshouse.itemhandler.environment;
+
+public enum EnvironmentVariables {
```

Comment:
* Like the use of an `enum` for this purpose. Compact and clear. 👍  
* I suppose could even call it `RequiredEnvironmentVariables`?

Reply:
Good idea, will change that